web_widget_domain_v11/static/src/copied-js/domain_selector_dialog.js

```diff
@@ -0,0 +1,47 @@
+odoo.define("web.DomainSelectorDialog", function (require) {
```

Comment:
OK, I was talking about the suspicious name `copied-js`, that it seems to work like a copied-pasted version for references, but not the final JS.